src/com/owncloud/android/ui/activity/DrawerActivity.java

```diff
@@ -236,6 +202,12 @@ public boolean onNavigationItemSelected(MenuItem menuItem) {
                                         ManageAccountsActivity.class);
                                 startActivityForResult(manageAccountsIntent, ACTION_MANAGE_ACCOUNTS);
                                 break;
+                            case R.id.drawer_menu_feedback:
+                                openFeedback();
```

Comment:
These are brandeable options, so you should check `help_enabled` and `feedback_enabled` flags in setup.xml and hide or show the options in the side menu depending on that.